content/chapter-03/ApplicationMetrics.md

```diff
@@ -9,13 +9,13 @@
   * Availability percentage, e.g.: Percentage of uptime for interval of time
   * Concurrent users, e.g.: Concurrent user count distribution over a day
   * User satisfaction using [Apdex score](https://www.apdex.org/)
-  * [Throughput](https://www.cdnetworks.com/en/news/6-critical-web-application-performance-metrics-to-consider/4257). This is very tight to the application at hand. It should be something representing a business value on a unit of time. For example, in an integration application, how many entities were moved from one system to another per using of time.
+  * [Throughput](https://www.cdnetworks.com/en/news/6-critical-web-application-performance-metrics-to-consider/4257). This is very tight to the application at hand. It should be something representing a business value on a unit of time. For example, in an integration application, how many entities were moved from one system to another per unit of time.
   
   
-* System level metrics ([Resource]())
-  * CPU utilization, e.g.: CPU utilization percentage per second
-  * Memory utilization, e.g.: Peak memory usage, Percentage of memory in used per application
-  * Network utilization, e.g.: Network bandwith usage per second
+* System level metrics ([Resource](https://blog.heroix.com/blog/how-to-monitor-server-performance))
```

Comment:
Resource => Reference and why not to hyperlink `System level metrics` directly?
Why double brackets?

Reply:
Take a look to see if it looks better. Parenthesis are more used to cite references, but they are more used on an article, not on an online material.